src/chrome/content/rules/Koding.xml

```diff
@@ -10,4 +10,7 @@
 	<rule from="^http://(\w+\.)?koding\.com/"
 		to="https://$1koding.com/" />
 
-</ruleset>
\ No newline at end of file
+
+  <exclusion pattern="^http://(?:www\.)?links\.koding\.com/" />
```

Comment:
You should include a `<test url="http://www.links.koding.com/" />` and one for `links.koding.com`.